### ISSUE TITLE
how to alter predefined network architecture and reuse the pretrained model please?

### ISSUE THREAD

**Issue by githubgsq** [tensorflow/tensorflow#19522]
hi，all , I'm faced with a problem.
I predefined a neural network architecture, simply, conv->pooling->conv->pooling->FC->softmax, then I trained the network with tensorflow also obtained checkpoint and models in directory: /mnist_convnet_model
Next, I want to reload the model. This time I need to alter the network's architecture, but still, weights pretrained needed to be reloaded.  The pretrained weights and the new nerual architecture do not match exactly but part of previous trained info is needed.
What should I do? Is there any API available please ?
Thx!

**Comment by achalshah20** [comment]
Just use **get_tensor_by_name** to get tensors from your graph. If you want add some graph, you can always mix existing tensors with the new ones.

**Comment by githubgsq** [comment]
hi, @achalshah20 ,thanks a lot for your advice! I would try it!!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatatodd: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
